docs/reference/indices/flush.asciidoc

```diff
@@ -101,7 +101,8 @@ which returns something similar to:
                      "translog_generation" : "2",
                      "max_seq_no" : "-1",
                      "sync_id" : "AVvFY-071siAOuFGEO9P", <1>
-                     "max_unsafe_auto_id_timestamp" : "-1"
+                     "max_unsafe_auto_id_timestamp" : "-1",
+                     "min_retained_seq_no": "0"
```

Comment:
why was this added?

Reply:
We [store](https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/index/engine/InternalEngine.java#L2318) the min_retained_seq_no of SoftDeletesPolicy in a Lucene index commit.

Reply:
But this is not part of this pr?

Reply:
Ah, this change is a part of this PR. We bake this value iff soft-deletes is enabled. Previously, we did not enable soft-deletes but now we do by default; therefore a Lucene commit also by default has it.